docs/admin/customize.rst

```diff
@@ -43,23 +42,22 @@ The best approach is to create a proper Python package out of your customization
         setup(
             name = "weblate_customization",
             version = "0.0.1",
-            author = "Michal Cihar",
-            author_email = "michal@cihar.com",
+            author = "Your name",
+            author_email = "yourname@example.com",
```

Comment:
Maybe do @orangesunny, or someone not @nijel that has written a customization that made it upstream?